bitcoinspv/chunk_test.go

```diff
@@ -2,37 +2,22 @@ package bitcoinspv
 
 import (
 	"testing"
-	"time"
 
-	"github.com/btcsuite/btcd/wire"
 	"github.com/gonative-cc/relayer/bitcoinspv/types"
 	"github.com/stretchr/testify/assert"
 )
 
-func createTestIndexedBlocks(t *testing.T, count int64, startHeight int64) []*types.IndexedBlock {
-	t.Helper()
-	blocks := make([]*types.IndexedBlock, count)
-	for i := int64(0); i < count; i++ {
-		hdr := wire.BlockHeader{Version: int32(i + 1), Timestamp: time.Now().Add(time.Duration(i) * time.Minute)}
-		blocks[i] = &types.IndexedBlock{
-			BlockHeight: startHeight + i,
-			BlockHeader: &hdr,
-		}
-	}
-	return blocks
-}
-
 func TestToBlockHeaders(t *testing.T) {
 	t.Run("single block", func(t *testing.T) {
-		inputBlocks := createTestIndexedBlocks(t, 1, 100)
+		inputBlocks := types.CreateTestIndexedBlocks(t, 1, 100)
 		headers := toBlockHeaders(inputBlocks)
 		assert.NotNil(t, headers)
 		assert.Len(t, headers, 1)
 		assert.Equal(t, inputBlocks[0].BlockHeader.Version, headers[0].Version)
 	})
 
 	t.Run("multiple blocks", func(t *testing.T) {
-		inputBlocks := createTestIndexedBlocks(t, 3, 200)
+		inputBlocks := types.CreateTestIndexedBlocks(t, 3, 200)
```

Comment:
It is unclear whether we should export the test function as public. Should we do this in test package?

Reply:
We could to this in test but then we cannot re-use it in `chunk_test.go` and `submit_test.go`. We export it publicly so we do not have to repeat code. (DRY).